tools/install/operator.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+
+## == Constant Definitions ==##
+TAB = " " * 4
+
+top_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))
+
+def guess_solib_dir():
+    dirs = glob.glob(os.path.join(top_dir, "bazel-bin/_solib_*"))
+    assert len(dirs) == 1, "Only one _solib_* directory under bazel-bin is expected"
```

Comment:
Don't know why, but It works.

Need to figure out the reason why it works in the next few days.

Reply:
Done.